src/test/java/apoc/export/util/JsonResults.java

```diff
@@ -0,0 +1,490 @@
+package apoc.export.util;
+
+import apoc.util.Util;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.joda.time.DateTime;
+import org.joda.time.LocalDateTime;
+
+import java.util.*;
+
+public class JsonResults {
+
+    public static Map<String, Object> nodeFoo(boolean datatypes) throws JsonProcessingException {
+
+        Map<String, Object> map = Util.map();
```

Comment:
Per questa cosa? Il metodo util prende <k, v> va utilizzato nel modo corretto:
```suggestion
        Map<String, Object> map = Util.map("id", 0, "type", "node", "labels", Arrays.asList("User"), "properties", Util.map("born", "2015-07-04T19:32:24", "name", "foo"));
```
e così via...
Inoltre non indentare mai il codice in questo modo, piuttosto mettere dei commenti che indichino che è un figlio.
Modificare tutto, e rimuovere tutta la gestione dei dataTypes per ora (Michael vuole aspettare che finalizziamo la modalità nel progetto Streams per riportarlo qui dentro)